diff --git a/OCOVR/openvr_api.cpp b/OCOVR/openvr_api.cpp
index 9cee799..acf561e 100644
--- a/OCOVR/openvr_api.cpp
+++ b/OCOVR/openvr_api.cpp
@@ -344,6 +344,11 @@ VR_INTERFACE uint32_t VR_CALLTYPE HmdSystemFactory()
 	OOVR_ABORT("HmdSystemFactory called - this is only here to make Proton recognise us, should never be called! Please file a bug, this is interesting.");
 }
 
+VR_INTERFACE void* VR_CALLTYPE VRControlPanel()
+{
+	return nullptr;
+}
+
 VR_INTERFACE bool VR_CALLTYPE VR_IsInterfaceVersionValid(const char* pchInterfaceVersion)
 {
 	return true; // Kinda dodgy
